Tools/Scripts/webkitpy/w3c/test_exporter_unittest.py

```diff
@@ -139,7 +139,7 @@ def test_export(self):
             'reset hard origin/master',
             'checkout new branch wpt-export-for-webkit-1234',
             'apply_mail_patch patch.temp -3',
-            'commit -a -m WebKit export of https://bugs.webkit.org/show_bug.cgi?id=1234',
+            'commit -a -m WebKit export: my bug title\r\n\r\nhttps://bugs.webkit.org/show_bug.cgi?id=1234',
```

Comment:
You forgot to update the newlines here.

Reply:
Yes, I need to squash sam's commit and update the tests.
I'd also like to understand the state of EWS webkitpy bots. @JonWBedard, are they supposed to be working?

Reply:
Done